client/fn-internal.h

```diff
@@ -138,6 +145,8 @@ struct s_Channel
   char *ttlforever_suffix;
   int client_cache_maxsize;
   void *serverrsakey[FRONTIER_MAX_SERVERN];
+  fn_query_stat query_stat;
+  int query_bytes;
```

Comment:
This at least could theoretically overflow at 2GB.  Would it be better to do `uint64_t`?

Reply:
2GB is awfully big for a single frontier query, but since values in s_FrontierStatisticNum are unsigned int, I'll change this to unsigned int so it can be up to 4GB.